adapters/adnuntius/adnuntius.go

```diff
@@ -124,8 +125,12 @@ func makeEndpointUrl(ortbRequest openrtb2.BidRequest, a *adapter, noCookies bool
 	tzo := -offset / minutesInHour
 
 	q := uri.Query()
-	if gdpr != "" && consent != "" {
+	if gdpr != "" {
+		endpointUrl = "https://europe.delivery.adnuntius.com/i"
```

Comment:
To avoid hardcoded constants like this we recommend to use `adapters.{bidder}.extra_info` config. 
Usage example: 

Config: https://github.com/prebid/prebid-server/blob/59dd6fbbb1d7efd92279b3724c3bea2c64763d16/config/config.go#L1019

Bidder code:
https://github.com/prebid/prebid-server/blob/59dd6fbbb1d7efd92279b3724c3bea2c64763d16/adapters/medianet/medianet.go#L85

Reply:
Oh, you can add Json in there! that's really good to know! :D Anyways the hardcoded stuff is now fixed. :)

Reply:
Thank you @mikael-lundin. There is a code conflict because we just merged in a PR which relocates adapter config from config.go into the yaml files. To resolve, please undo:

config.go:
```
v.SetDefault("adapters.adnuntius.extra_info", "https://europe.delivery.adnuntius.com/i")
```

... and add ...

adnuntius.yaml:
```
extra_info: "https://europe.delivery.adnuntius.com/i"
```

You may need to merge in the changes from master first. Sorry for the less than ideal timing on this change.

Reply:
You still need to do this. The merge conflict is blocking the GitHub test runner.